CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
 - Add: support for APIv2 notifications (POST /v2/op/notify), which opens up for true APIv2 federation (#2986)
 - Fix: bug having entities without attibutes AND matching registrations made the attr-less entity not be returned by GET /v2/entities (#3176)
 - Fix: bug when using "legacy" format for custom v2 notifications (#3154)
+- Fix: hardwired HTTP header strings by replacing them with macros in HttpHeaders.h
```

Comment:
No real impact from a final user perspective... we sometimes ommit this kind of changes in CNR or, if the change is big and we want to have record of it, use "Hardening:" instead of "Fix:". Both options are ok in this case, chose whichever you prefer :)

(If you keep it at the end, please include the issue number in the CNR line)

Reply:
CNR line is removed